frontend/src/views/MyProgress/index.vue

```diff
@@ -163,6 +200,7 @@ export default {
           this.canteen?.centralKitchenDiagnostics?.find((x) => +x.year === +this.year)
         )
       }
+      this.centralKitchenDiagnosticMode = this.diagnostic?.centralKitchenDiagnosticMode
     },
     assignTab() {
```

Comment:
Dernière remarque... Les tabs grisés (inactives) sont toujours accessibles depuis l'historique et la navigation du browser (vidéo ci-dessous).

Je pense qu'ici dans `assignTab` on peut dire : si `this.centralKitchenDiagnosticMode === "APPRO"` et que le tab n'est ni `appro` ni `établissement` on fait un `this.$router.replace({ name: "MyProgress", params: { measure: approId } })`. Qu'est-ce que t'en penses ?

[Screencast from 20-10-23 11:25:08.webm](https://github.com/betagouv/ma-cantine/assets/1225929/5407a38e-1f41-4641-8d17-16f6467a0114)